src/source/examples/metrics-query-v2.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
what's the point of duplicating the example if you're not even showing the actual new fields?